docs/NETWORK-ACLS.md

```diff
@@ -0,0 +1,289 @@
+# Network ACLs
+
+An access control list (ACL) consists of one or more access control entries (ACEs) that collectively define
+the network traffic profile.
+This document describes different types of ACEs, their semantics, limitations and provides few basic examples
+for common use-cases.
+
+## Configuration Model
+
+In EVE, ACLs are configured separately for every ECO interface.
+Protobuf-modeled configuration model of ACLs can be found in the [fw.proto file](../api/proto/config/fw.proto).
+To control the network traffic between a given ECO and other endpoints within the scope of a given network instance,
+add one or more ACEs under the `acls` field of [NetworkAdapter](../api/proto/config/netconfig.proto).
+
+Every ACE should have a unique non-zero integer ID assigned. Please note that ACE ID should not exceed 24bits.
+ACE ID is set in [flowlogs](../api/proto/flowlog/flowlog.proto) as `aclId` for every flow to show which ACE was applied.
+ACE `ID=0` is reserved for a default reject-all rule that implicitly exists at the end of every ACL (even flows whose
+packets are rejected are logged).
+
+## ACL Semantics
+
+Every ECO interface is assigned a single separate access control list filtering inbound and outbound traffic
+(unless it is a passthrough interface, in which case EVE networking is completely bypassed).
```

Comment:
I think we use the term "app-direct" in the API and not "passthrough"

Reply:
fixed

Reply:
Actually one comment says app-direct. The enum value is called PhyIoUsageDedicated. Sigh.

Reply:
updated again :)